libgstc/libgstc.c

```diff
@@ -365,6 +365,46 @@ gstc_client_ping (GstClient * client)
   return gstc_cmd_send (client, request);
 }
 
+GstcStatus
+gstc_client_debug(GstClient *client, const int threshold, const int colors)
+{
+  GstcStatus ret;
+  char* how;
+  char* how_fmt = "*:%d";
+  
+  gstc_assert_and_ret_val (NULL != client, GSTC_NULL_ARGUMENT);
+
+  //Enable debug of the pipeline
+  ret = gstc_cmd_update (client, "/debug/enable", "true");
```

Comment:
We need to find a way to disable debug. Is passing a 0 threshold the same as using `/debug/enable false?`

Reply:
I think yes, using 0 would be equivalent to setting the flag as false